src/App.css

```diff
@@ -5,38 +5,14 @@
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
-}
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
-}
-
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.form-label {
```

Comment:
nice to add styles like this. a useful resource could be [BEM methodology](https://css-tricks.com/bem-101/),

Another thing, if the class name was added only to modify one CSS attribute, you can also consider having utility classes, where the class name corresponds to the attribute being modified i.e.
```
.mr_10 {
  margin-right: 10px;
}
```